www/src/content/blog/2025-02-18-otel-cloudflare-worker.mdx

```diff
@@ -0,0 +1,146 @@
+---
+title: "An OpenTelemetry collector in Rust for Cloudflare Workers"
+description: "We’re excited to introduce an open source OpenTelemetry collector service, specifically for Cloudflare Workers."
+slug: otel-cloudflare-worker
+date: 2025-02-18
+author: Micha "mies" Hernandez van Leuffen
+tags:
+  - Cloudflare
+  - OpenTelemetry
+  - Rust
+---
+
+import PackageManagers from "@/components/PackageManagers.astro";
+import { LinkCard } from "@astrojs/starlight/components";
+import playgroundVideo from "../../assets/blog/2025-02-12-playground.mov";
+
+In today's distributed systems landscape, observability is crucial for understanding 
+and debugging complex applications. Today, I'm excited to share details about 
+[otel-worker](https://github.com/fiberplane/otel-worker). Think of it as an OpenTelemetry collector, 
+but without the YAML complexity and heavy infrastructure.
+
+We built otel-worker after encountering some persistent latency with Cloudflare’s D1,
+where the real bottleneck was elusive. 
+We wanted a straightforward, powerful way to capture, store, and analyze our 
+production traces. 
+
+There are existing OpenTelemetry observability solutions including Datadog (if you enjoy throwing money at problems), Baselime, and open source projects, but we decided to roll our own, written in 
+Rust and specifically geared towards Cloudflare Workers. This has some key benefits:
+
+- Traditional OpenTelemetry collectors require significant infrastructure
+- Why process teletetry in `us-east-1` when you can can store them directly at the edge with Cloudflare’s D1?
```

Comment:
nit: cloudflare has not yet implemented read replicas for D1, so the data is not really at the edge 

also typo: `teletetry`